<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="meatSharedTableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>4</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="groupSummaryTableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>4</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="meatSharedLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jaetut lihat</string>
    </property>
   </widget>
   <widget class="QLabel" name="groupSummaryLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jakoryhmien yhteenveto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshPushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Päivitä</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
